src/js/containers/BrowserCacheTTLCard/BrowserCacheTTLCard.js

```diff
@@ -14,7 +14,7 @@ class BrowserCacheTTLCard extends Component {
 
     handleChange(value) {
         let { dispatch } = this.props;
-        dispatch(asyncZoneUpdateSetting(SETTING_NAME, this.props.activeZoneId, value));
+        dispatch(asyncZoneUpdateSetting(SETTING_NAME, this.props.activeZoneId, value.value));
```

Comment:
same here let's change the variable name for everything in 4505811f183f721e48da108a0d0e5d5dcf7ea186